python/geoarrow-core/src/interop/shapely/from_shapely.rs

```diff
@@ -76,98 +76,107 @@ fn call_to_wkb<'a>(
 }
 
 #[pyfunction]
-#[pyo3(signature = (input, *, crs = None))]
+#[pyo3(signature = (input, *, crs = None, method = "wkb"))]
 pub fn from_shapely(
     py: Python,
     input: &Bound<PyAny>,
     crs: Option<PyCrs>,
+    method: String,
```

Comment:
Here since this is enum input, we want a validated parameter, not a bare string, like this:

https://github.com/geoarrow/geoarrow-rs/blob/5921e42d17f4d385a68de17cda6d4e2884e7cfe6/rust/pyo3-geoarrow/src/coord_type.rs#L6-L22